labelbox/schema/dataset.py

```diff
@@ -58,12 +60,75 @@ class Dataset(DbObject, Updateable, Deletable):
         deprecation_warning=
         "This method does not return any data for batch-based projects and it will be deprecated on or around November 1, 2023."
     )
-    data_rows = Relationship.ToMany("DataRow", False)
     created_by = Relationship.ToOne("User", False, "created_by")
     organization = Relationship.ToOne("Organization", False)
     iam_integration = Relationship.ToOne("IAMIntegration", False,
                                          "iam_integration", "signer")
 
+    def data_rows(
+        self,
+        from_cursor: str = None,
+        where: Comparison = None,
+    ) -> PaginatedCollection:
+        """ 
+        Custom method to paginate data_rows via cursor.
+
+        Params:
+            after (str): Cursor (data row id) to start from, if none, will start from the beginning
+            where (dict(str,str)): Filter to apply to data rows. Where value is a data row column name and key is the value to filter on.    
+                example: {'external_id': 'my_external_id'} to get a data row with external_id = 'my_external_id'
+
+
+        NOTE: 
+            Order of retrieval is newest data row first.
+            Deleted data rows are not retrieved.
+            Failed data rows are not retrieved.
```

Comment:
We are returning both failed and processing data rows too.
Code: https://github.com/Labelbox/intelligence/blob/a688618adaccc8eec391ea479535d94fd15944ce/apps/api/src/catalog/adv/data-rows/data-rows-router.service.ts#L159 (the comment was not removed after all)

Reply:
oh ok that description is for mysql.. I did not want to call out for ADV vs mysql specifically here... @kkim-labelbox pls advice which way I should describe this

Reply:
but I have looked at it again and the thing is, we have a const `_PAGE_SIZE` explicitly setting page size in `PaginatedCollection`. So, I guess I could add some param to '`not_set_page_size` and use it for this particular case... I know though we have a separate story to review pagination so perhaps I do it there?